test-helpers/src/helpers.rs

```diff
@@ -45,3 +86,99 @@ pub fn provide_liquidity<'a, R: Runner<'a>>(
         )
         .unwrap();
 }
+
+pub fn cw20_mint<'a, R: Runner<'a>>(
+    runner: &'a R,
+    cw20_addr: String,
+    recipient: String,
+    amount: Uint128,
+    signer: &SigningAccount,
+) -> RunnerExecuteResult<MsgExecuteContractResponse> {
+    let wasm = Wasm::new(runner);
+    wasm.execute(
+        &cw20_addr,
+        &Cw20ExecuteMsg::Mint { recipient, amount },
+        &[],
+        signer,
+    )
+}
+
+pub fn cw20_transfer<'a, R: Runner<'a>>(
+    runner: &'a R,
+    cw20_addr: String,
+    recipient: String,
+    amount: Uint128,
+    signer: &SigningAccount,
+) -> RunnerExecuteResult<MsgExecuteContractResponse> {
+    let wasm = Wasm::new(runner);
+    wasm.execute(
+        &cw20_addr,
+        &Cw20ExecuteMsg::Transfer { recipient, amount },
+        &[],
+        signer,
+    )
+}
+
+/// Query the balance of a cw20 token
+pub fn cw20_balance_query<'a>(
+    runner: &'a impl Runner<'a>,
+    cw20_addr: String,
+    address: String,
+) -> StdResult<Uint128> {
+    let res: BalanceResponse = Wasm::new(runner)
+        .query(&cw20_addr, &Cw20QueryMsg::Balance { address })
+        .unwrap();
+
+    Ok(res.balance)
+}
+
+/// Query the balance of a native token
+pub fn query_token_balance<'a, R>(runner: &'a R, denom: &str, address: &str) -> Uint128
+where
+    R: Runner<'a>,
+{
+    let bank = Bank::new(runner);
+    let balance = bank
+        .query_balance(&QueryBalanceRequest {
+            address: address.to_string(),
+            denom: denom.to_string(),
+        })
+        .unwrap()
+        .balance
+        .unwrap_or_default()
+        .amount;
+    Uint128::from_str(&balance).unwrap()
+}
```

Comment:
there is a helper for this in cw-it called `bank_balance_query` or something along those lines

Reply:
changed to use that one.